app/src/main/java/com/thinqtv/thinqtv_android/MainActivity.java

```diff
@@ -32,10 +37,19 @@ protected void onCreate(Bundle savedInstanceState) {
                 .setWelcomePageEnabled(false)
                 .build();
         JitsiMeet.setDefaultConferenceOptions(defaultOptions);
+
+        // COMMENTED OUT BECAUSE I'M NOT SURE IF IT ACTUALLY LOOKS GOOD. GET TEAM OPINION
+        //mainViewer = (VideoView) findViewById(R.id.videoview);
+        //Uri uri = Uri.parse("android.resource://" + getPackageName() + "/" + R.raw.crowdpubcommercialv_1);
+        //mainViewer.setVideoURI(uri);
+        //mainViewer.start();
     }
 
+// COMMENTED OUT BECAUSE BUTTON IS NOT USEFUL ANYMORE
```

Comment:
Same here - just remove the code entirely, we have access to it in other branches and past commits.